js/views/SocialBtns.js

```diff
@@ -0,0 +1,63 @@
+import app from '../app';
+import loadTemplate from '../utils/loadTemplate';
+import { followedByYou, followUnfollow } from '../utils/follow';
+
+import BaseVw from './baseVw';
+
+export default class extends BaseVw {
+  constructor(options = {}) {
+    if (!options.targetID) throw new Error('You must provide a targetID');
+
+    super(options);
+    this.options = options;
+
+    this._state = {
+      following: followedByYou(options.targetID),
+      processing: false,
+      ...options.initialState || {},
```

Comment:
It may not be a bad idea to set default values for btnClasses and stripClasses since I imagine those might be fine for a lot of implementations. I would also probably set them as state, rather than options.